src/service/repo/signedMessageRepo.ts

```diff
@@ -39,6 +42,7 @@ interface SignedMessageRepoTypes {
   getSingleSignedPostNotifications: (activityId: string) => Promise<SignedPostNotification>;
   setChannelAsRead: (channelId: string, channelType: ChannelTypeEnum) => Promise<boolean>;
   createSignedChat: (body: string[]) => Promise<any>;
+  getSignedChannelDetail: (channelType: GetstreamChannelType, channelId: string) => Promise<any>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The return type of the `getSignedChannelDetail` function is marked as `Promise<any>`. It's a good practice to avoid using `any` when possible. If the structure of the data returned by this function is known, it would be better to define an interface for it and use that instead.
```diff
- getSignedChannelDetail: (channelType: GetstreamChannelType, channelId: string) => Promise<any>;
+ getSignedChannelDetail: (channelType: GetstreamChannelType, channelId: string) => Promise<ChannelDetailInterface>;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->